HuntTheWumpus/HuntTheWumpus/Player.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HuntTheWumpus
+{
+    class Player
+    {
+        public bool IsAlive { get; set; } // up for further debate
+        public Room CurrentLocation { get; set; }
+
+        public void ShootAt(int roomNumber)
+        {
+            if (Map.Rooms.ElementAt(roomNumber).Hazard is Wumpus)
+            {
+                Console.WriteLine("You are a winner! Congratulations!");
+            }
+            else
+            {
+                Console.WriteLine("You are a loser!  Game over!");
+            }
+        }
+
+        public Hazard MoveTo(int roomNumber)
```

Comment:
Question: Can the player be placed in a room with a hazard by the bat? If not, then recursion could be used to conduct the checks